file/deferred.go

```diff
@@ -0,0 +1,173 @@
+package file
+
+import (
+	"context"
+	"io"
+
+	dagpb "github.com/ipld/go-codec-dagpb"
+	"github.com/ipld/go-ipld-prime"
+)
+
+func newDeferredFileNode(ctx context.Context, lsys *ipld.LinkSystem, root ipld.Link) LargeBytesNode {
+	dfn := deferredFileNode{
+		LargeBytesNode: nil,
+		root:           root,
+		l:              lsys,
+		ctx:            ctx,
+	}
+	dfn.LargeBytesNode = &deferred{&dfn}
+	return &dfn
+}
+
+type deferredFileNode struct {
+	LargeBytesNode
+
+	root ipld.Link
+	l    *ipld.LinkSystem
```

Comment:
I would persoanlly choose a longer name, because `d.l` 100 lines further down in the file will not mean anything to most readers :) how about `lsys`?

Reply:
Or even personally, if I could spell.